src/app/(post-login)/submission/components/ProgramMenu.tsx

```diff
@@ -55,26 +66,29 @@ const ProgramMenu = ({ shortNameSearchQuery }: { shortNameSearchQuery: string })
 	const params = useParams();
 	const pathname = usePathname();
 	const router = useRouter();
+	const { permissions, egoJwt } = useAuthContext();
+	const { DATA_CENTER } = useAppConfigContext();
 
 	// params can be arrays from dynamic routing
 	const activeProgramName = typeof params.shortName === 'string' ? params.shortName : '';
 
-	const { data: programsData, loading, error } = useQuery(SIDEMENU_PROGRAMS);
+	const {
+		data: programsData,
+		loading,
+		error,
+	} = useQuery(SIDEMENU_PROGRAMS, { variables: { dataCenter: DATA_CENTER } });
 	const programs = programsData?.programs || [];
 
 	const filteredPrograms = !shortNameSearchQuery.length
-		? programs
-		: programs.filter(
-				(program) =>
-					program && program.shortName.search(new RegExp(shortNameSearchQuery, 'i')) > -1,
-		  );
-
-	const setActiveProgram =
-		(shortName: string): MouseEventHandler =>
-		() => {
-			const url = `/submission/program/${shortName}/registration`;
-			router.push(url);
-		};
+		? programs.filter(notNull)
+		: programs
+				.filter(notNull)
+				.filter((program) => program.shortName.search(new RegExp(shortNameSearchQuery, 'i')) > -1);
+	const sortedProgramList = orderBy(filteredPrograms, 'shortName');
+
+	const [activeProgram, setActiveProgram] = useState(activeProgramName);
```

Comment:
Program dropdown was not closing which is why I've added this

Reply:
https://github.com/icgc-argo/rdpc-ui/pull/115/files#diff-b1be0f38440ac03e9ef18232ac95c5f998fe4162a630b28ff29d72cc538e7d1c 

I moved the state to the url in the above PR

I think there's likely a way we can fix the dropdown without reverting back to "local" state ie. `useState`

Reply:
Discussing on Slack; there needs to be separation between opening/closing menu items and navigating to a specific page